dist/php/businessEmailValidator.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+class BusinessEmailValidator
+{
+
+	/**
+	* Used to load list of blocked domains from local JSON file.
+	* @access private
+	* @return array
+	*/
+	private function listBlocked()
+	{
+		$path = 'src/freeEmailService.json';
+		file_exists($path) or die("Error: file 'freeEmailService.json' not found!");
```

Comment:
Die is a bit hard here ? Maybe a Throw is a better solution to catch the error ?